tests/crypto/test_cases/CMakeLists.txt

```diff
@@ -11,14 +11,15 @@ zephyr_linker_sources(RODATA
     ${CMAKE_CURRENT_SOURCE_DIR}/custom-rodata.ld
 )
 
+if(NOT CONFIG_CRYPTO_SKIP_HASH_TESTS)
```

Comment:
Please indent the lines between this `if()` and the following `endif()`

Reply:
Indented